seed/views/v3/data_quality.py

```diff
@@ -10,124 +10,40 @@
 from celery.utils.log import get_task_logger
```

Comment:
What do you think about renaming this file to data_quality_checks.py?
I think thus far our files have been following the plural form of the model.